app/src/app/content-plugin/ui/custom-theme.scss

```diff
@@ -2,6 +2,7 @@
 @import './overrides/adf-style-fixes.theme';
 @import './overrides/adf-pagination.theme';
 @import './overrides/ay11';
+@import '../../ui/overrides/adf-about.theme.scss';
```

Comment:
this file should be moved to the same location as other in this import section

Reply:
moved the 'adf-about.theme.scss' style override file along with other overrides and deleted unwanted/empty folders (app/src/app/ui/overrides).